jsonschema-validation.xml

```diff
@@ -801,16 +802,31 @@
 
             <section title="Implementation Requirements">
                 <t>
-                    The content keywords function as both annotations and as assertions.
-                    While no special effort is required to implement them as annotations conveying
-                    how applications can interpret the data in the string, implementing
-                    validation of conformance to the media type and encoding is non-trivial.
+                    Due to security and performance concerns, as well as the open-ended nature of
+                    possible content types, implementations MUST NOT automatically decode, parse,
+                    and/or validate the string contents by default.  This additionally supports
+                    the use case of embedded documents intended for processing by a different
+                    consumer than that which processed the containing document.
+                </t>
+                <t>
+                    All keywords in this section apply only to strings, and have no
+                    effect on other data types.
                 </t>
                 <t>
-                    Implementations MAY support the "contentMediaType" and "contentEncoding"
-                    keywords as validation assertions.
-                    Should they choose to do so, they SHOULD offer an option to disable validation
-                    for these keywords.
+                    Implementations MAY offer the ability to decode, parse, and/or validate
+                    the string contents automatically.  However, it MUST NOT perform these
+                    operations by default, and MUST provide the validation result of each
+                    string-encoded document separately from the enclosing document.
```

Comment:
I see the comment below, but I'm curious what this would look like if someone expects to get multiple schema validation result objects from a single `validate` call.  At least, that's what I would expect if I were consuming an implementation and I read that.

I think a note saying that it would/could require separate validation passes to validate the contents would be nice.

Reply:
@gregsdennis OK see if the new wording works for you.

Reply:
That looks better.